src/main/java/org/prebid/server/auction/requestfactory/Ortb2ImplicitParametersResolver.java

```diff
@@ -84,6 +86,7 @@
     public Ortb2ImplicitParametersResolver(boolean shouldCacheOnlyWinningBids,
```

Comment:
Why have you reverted parameter and field name changes `shouldCacheOnlyWinningBids` -> `cacheOnlyWinningBids`?